### PR TITLE
created gallery page

### PR DESCRIPTION
**Pull request by DevanshXerc90**

### CODE REVIEW

File: public/g6.jpg
Comment (singh-ab) [inline public/g6.jpg:1]: Add a more appropriate photo

**Review by singh-ab — COMMENTED** [review]
Use a better image layout style, the vertical image layout seems good enough for mobile mode but doesn't look good in desktop mode. Contact @BlackEyes09 for more detailed reviews.

**Comment by DevanshXerc90** [comment]
Okay, working on it.

**Comment by BlackEyes09** [comment]
Try to make it more like a grid view. collapsable accordingly with mobile view.

**Pull request closed without merging**